sample/programs/cpp/perf-script.sh

```diff
@@ -14,6 +14,14 @@ if [[ "$1" == "forks" ]]; then
   exit 0
 fi
 
+if [[ "$1" == "idle" ]]; then
+  perf record -a -F 999 -g ./idle > perf.data
```

Comment:
```suggestion
  perf record -a -F 999 -g --output=perf.data ./idle
```

The same applies below and above. These commands aren't printing anything yet, but perhaps it would be worth adding this flag instead as if in the future they print something, `perf script` and related tooling will break and it will be a bit confusing 😄

Reply:
I needed to do something very slightly fancier here, because the stdout of `perf-script.sh` is used directly to generate the output files in the `Makefile`, but this `--output` switch was still a helpful key